controller/api/models.py

```diff
@@ -519,7 +536,6 @@ class Push(UuidAuditedModel):
     class Meta:
         get_latest_by = 'created'
         ordering = ['-created']
-        unique_together = (('app', 'uuid'),)
```

Comment:
It's just that this changes the schema definition:
```sql
CREATE TABLE "api_push" (
    "uuid" uuid NOT NULL PRIMARY KEY,
    "created" timestamp with time zone NOT NULL,
    "updated" timestamp with time zone NOT NULL,
    "owner_id" integer NOT NULL,
    "app_id" uuid NOT NULL REFERENCES "api_app" ("uuid") DEFERRABLE INITIALLY DEFERRED,
    "sha" varchar(40) NOT NULL,
    "fingerprint" varchar(255) NOT NULL,
    "receive_user" varchar(255) NOT NULL,
    "receive_repo" varchar(255) NOT NULL,
    "ssh_connection" varchar(255) NOT NULL,
    "ssh_original_command" varchar(255) NOT NULL,
    UNIQUE ("app_id", "uuid")
)
;
```
That last `UNIQUE` constraint / index is no longer in the table definition--ditto for two other tables. So now the database won't guarantee that there isn't an `api_push` table entry with duplicates for the `app` foreign key and the `uuid` field. I think that should remain in the schema definition. (And if not, we should provide a Django migration to alter those tables and drop the constraint.)